.pre-commit-config.yaml

```diff
@@ -10,7 +10,7 @@ repos:
         args: ['--target-version', 'py37']
   - repo: https://github.com/timothycrosley/isort
     # isort config is in setup.cfg
-    rev: 5.6.4
+    rev: 5.12.0
```

Comment:
Can this be done as a separate PR, just avoids noise on this PR.

I will remove this change when rebasing.